app/javascript/packages/prompt-on-navigate/index.ts

```diff
@@ -25,8 +19,6 @@ export function promptOnNavigate(options: PromptOnNavigateOptions = defaults): (
     ev.preventDefault();
     ev.returnValue = '';
 
-    trackEvent(PROMPT_EVENT, { path: window.location.pathname });
-
     const stillOnPageIntervalsInSeconds = [...options.stillOnPageIntervalsInSeconds];
```

Comment:
Can probably remove everything in this function after `ev.returnValue = '';` and support for `stillOnPageIntervalsInSeconds`

Reply:
Do we need this file at all? I'm looking at #8512 and it was newly added there, which seems to imply we don't need it if we're logging anymore?

(I'm sorta viewing this PR as essentially a revert of #8512)